tracing-error/src/heap_error.rs

```diff
@@ -0,0 +1,81 @@
+use crate::SpanTrace;
+use std::error::Error;
+use std::fmt::{self, Debug, Display};
+
+///
```

Comment:
Before this is merged, we should definitely replace all of these empty doc comments with "real" docs...